File: Java Script - Introduction to Programming/7.1. Complex Loops/15. Number Table.js
function numberTable(n) {
  let result = ''

  for (let i = 1; i <= n; i++) {
    for (let j = 0; j < n; j++) {
      let s = i + j

      if (s > n) {
        s = 2 * n - s
      }
      result += s + ' '
    }

    console.log(result)
    result = ''
  }

}

// numberTable(6)

File: Pyrhon - Introduction to Programming/7.1. Complex Loops/15. Number Table.py
n = int(input())
result = ''

for i in range(1, n + 1):
    for j in range(n):
        s = i + j
        if s > n:
          s = 2 * n - s

        result += str(s) + ' '

    print(result)
    result = ''

